aiven/client/cli.py

```diff
@@ -4037,6 +4039,13 @@ def service__create(self) -> None:
                         "source_service": self.args.read_replica_for,
                     }
                 )
+        elif self.args.recovery_target_time and self.args.source_service_name:
+            if self.args.service_type in ("pg", "myqsl"):
```

Comment:
If `recovery_target_time` is valid only for some service types, then I think `service create` command should fail if it's given for a service of some other type. This way, we don't create a service in a state other than what the user is expecting. Better for them to be informed that recovery to a specific time is not possible for the forked service.

Reply:
I am thinking maybe we let this fail at the `api` so that incase we add support for PITR for another service besides `pg` and `mysql` then we don't have to come and make a change here. Opinion on this?

Reply:
Yes, that would be my preference.